### PR TITLE
Analysis of phantom zone flag

### PR DESCRIPTION
**Pull request by tainn1**

### CODE REVIEW

File: _documentation/user/flags/phantom-zone.md [hunk 1]
```diff
@@ -5,4 +5,12 @@ type: Good
 summary: Teleporting toggles Zoned effect. Zoned tank can drive through buildings. Zoned tank shoots Zoned bullets and can't be shot (except by superbullet, shock wave, and other Zoned tanks).
 ---
 
-...
+The Phantom Zone (PZ) flag allows the player to enter the Zoned state when passing through a teleporter. Exceptionally, passing through such a teleporter while holding the flag does not trigger an actual positioning teleport regardless of link settings, only applying the Zoned effect. Passing through a teleporter while Zoned reverts the tank back to its original state.
```
[the remark this answers is not in the document]
Comment (tainn1) [reply _documentation/user/flags/phantom-zone.md]: @jwmelto I would ask you to give a concrete alternative. It is major functional info and I would require you to form something up, considering you seem to dislike and not presenting how or why.

File: _documentation/user/flags/phantom-zone.md [hunk 2]
```diff
@@ -5,4 +5,12 @@ type: Good
...
+
+Zoned tanks can drive through buildings, but unlike tanks with the [Oscillation Overthruster](../oscillation-overthruster/) flag, Zoned tanks can also drive backwards and shoot while inside buildings.
+
+Whiled Zoned, the tank shoots Zoned bullets. Such bullets are only capable of killing other Zoned tanks. On the other hand, Zoned tanks can also be killed by tanks holding [Super Bullet](../super-bullet/) or [Shock Wave](../shock-wave/) flags.
+
+## Tactics
+
+Phantom Zone can be utilized as a means of quick navigation or for escaping purposes. The former is viable when attempting to directly reach a certain destination, such as the opponent's team flag, a zone filled with desirable flags, or a viable battle position. The latter can be considered at the rabbit hunt game mode to escape as a rabbit for an extended period of time.
```
[the remark this answers is not in the document]
Comment (tainn1) [reply _documentation/user/flags/phantom-zone.md]: @jwmelto no, the point of rabbit hunt is for hunters to kill the rabbit and the rabbit to stay the rabbit as long as possible, that's how the game mode is usually played.

[the remark this answers is not in the document]
Comment (jwmelto) [reply _documentation/user/flags/phantom-zone.md]: sorry, I misread the sentence. "escape as a rabbit" is viable.

File: _documentation/user/flags/phantom-zone.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwmelto) [reply _documentation/user/flags/phantom-zone.md]: Delete the sentence entirely, and it reads just fine. Like I said, I don't know what the point of the sentence is.

[the remark this answers is not in the document]
Comment (tainn1) [reply _documentation/user/flags/phantom-zone.md]: That is a possibility, but don't you believe it is relatively major functional info (for a newcomer, for example) knowing that passing a teleporter while holding the PZ flag will not trigger the actual teleport?

[the remark this answers is not in the document]
Comment (tainn1) [reply _documentation/user/flags/phantom-zone.md]: @jwmelto sentence deleted and patch pushed.

**Pull request merged**